_includes/nav-bar.html

```diff
@@ -36,22 +70,49 @@
             </div>
           </li>
           <li class="nav-item dropdown">
-            <a class="nav-link dropdown-toggle" href="/documentation/" id="navbarDropdownMenuLink" data-toggle="dropdown"
-              aria-haspopup="true" aria-expanded="false">
+            <a
+              class="nav-link dropdown-toggle"
+              href="/documentation/"
+              id="navbarDropdownMenuLink"
+              data-toggle="dropdown"
+              aria-haspopup="true"
+              aria-expanded="false"
+            >
               Docs
             </a>
             <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
-              <a class="dropdown-item" target="_blank" href="https://ros-planning.github.io/moveit_tutorials/">Tutorials</a>
-              <a class="dropdown-item" href="/documentation/concepts/">Concepts</a>
-              <a class="dropdown-item" href="/documentation/related_projects/">Related Projects</a>
-              <a class="dropdown-item" href="/documentation/plugins/">Plugin Interfaces</a>
-              <a class="dropdown-item" href="/documentation/planners/">Planners</a>
-              <a class="dropdown-item" href="/documentation/source-code-api/">Source Code & API</a>
+              <a
+                class="dropdown-item"
+                target="_blank"
+                href="https://ros-planning.github.io/moveit_tutorials/"
+                >Tutorials</a
```

Comment:
what about these? I thought we had some code format in place

Reply:
this is a very weird format, none of the other developers who have worked on this project have ever done it this way. including myself. i don't agree with this weird format

Reply:
it's not a weird format it's just max lenght I will fix it